libmproxy/web/app.py

```diff
@@ -284,8 +309,11 @@ def put(self):
 
 class Application(tornado.web.Application):
 
-    def __init__(self, master, debug):
+    def __init__(self, master, debug, wauthenticator):
         self.master = master
+        self.additional_args = dict(
```

Comment:
Can we use http://www.tornadoweb.org/en/stable/web.html#tornado.web.RequestHandler.settings instead of passing that manually to each RequestHandler?